index.js

```diff
@@ -284,8 +284,9 @@ ExtractTextPlugin.prototype.apply = function(compiler) {
 				if(extractedChunk.modules.length) {
 					extractedChunk.modules.sort(function(a, b) {
 						if(isInvalidOrder(a, b)) {
-							compilation.errors.push(new OrderUndefinedError(a.getOriginalModule()));
-							compilation.errors.push(new OrderUndefinedError(b.getOriginalModule()));
+							var targetLog = options.relaxInvalidOrder ? compilation.warnings : compilation.errors;
```

Comment:
This could go hand in hand with the `ignoreOrder` option, instead of adding a new option